implementations/rust/ockam/ockam_command/src/tcp/inlet/create.rs

```diff
@@ -239,36 +233,56 @@ impl Command for CreateCommand {
         cmd.add_inlet_created_event(&opts, &node_name, &inlet_status)
             .await?;
 
-        let created_message = fmt_ok!(
-            "Created a new {} in the Node {} bound to {}\n",
-            color_primary_alt(inlet_str),
+        let created_message = format!(
+            "Created a new TCP Inlet in the Node {} bound to {}\n",
             color_primary(&node_name),
-            color_primary(cmd.from.to_string())
+            color_primary(cmd.from.to_string()),
         );
 
+        let privileged_mode_str = fmt_info!(
+            "This Inlet is operating in {} mode\n",
+            color_primary_alt("privileged".to_string())
+        );
+
+        let created_message = fmt_ok!("{}", created_message);
+
         let plain = if cmd.no_connection_wait {
-            created_message
+            let mut msg = created_message
                 + &fmt_log!(
-                    "It will automatically connect to the {} at {} as soon as it is available",
-                    color_primary_alt(outlet_str),
+                    "It will automatically connect to the TCP Outlet at {} as soon as it is available\n",
                     color_primary(&cmd.to)
-                )
+                );
+
+            if privileged {
+                msg += &privileged_mode_str;
+            }
+
+            msg
         } else if inlet_status.status == ConnectionStatus::Up {
-            created_message
+            let mut msg = created_message
                 + &fmt_log!(
-                    "sending traffic to the {} at {}",
-                    color_primary_alt(outlet_str),
+                    "sending traffic to the TCP Outlet at {}\n",
                     color_primary(&cmd.to)
-                )
+                );
+
+            if privileged {
+                msg += &privileged_mode_str;
+            }
+
+            msg
         } else {
-            fmt_warn!(
-                "A {} was created in the Node {} bound to {} but failed to connect to the {} at {}\n",
-                color_primary_alt(inlet_str),
+            let mut msg = fmt_warn!("A TCP Inlet was created in the Node {} bound to {} but failed to connect to the TCP Outlet at {}\n",
                 color_primary(&node_name),
                 color_primary(cmd.from.to_string()),
-                color_primary_alt(outlet_str),
-                color_primary(&cmd.to)
-            ) + &fmt_info!("It will retry to connect automatically")
+                color_primary(&cmd.to));
+
+            msg += &fmt_info!("It will retry to connect automatically\n");
+
+            if privileged {
+                msg += &privileged_mode_str;
+            }
+
+            msg
         };
 
```

Comment:
You could move the privileged message out here:

```
let mut plain = { ... }
if privileged {
    plain += &privileged_mode_str;
}
```

Reply:
Nice catch, thank you